integration_tests/features/support/steps.js

```diff
@@ -902,8 +907,13 @@ Then(
       );
       this.lastResult = await client.transactionState(sig);
```

Comment:
We will wait a really long time here for a node that did not receive the transaction; suggest making the wait time here for `async () => client.transactionStateResult(sig)` a lot shorter and evaluate a test condition to decide if we should wait again. Sample code below (_I could not figure out query the total seeds and nodes so do not mind the first bit_):

``` rust
    let succeeded = 0;
    let total = 0;
    for (const seed in this.seeds) {
      total += 1;
    }
    for (const seed in this.nodes) {
      total += 1;
    }
    canFail = canFail ? parseInt(canFail) : 0;
    const needed = Math.floor(total * (1 - canFail / 100));
    await this.forEachClientAsync(async (client, name) => {
      for (let i = 0; i < 1100; i++) {
        if (succeeded < needed) {
          await waitFor(
            async () => client.transactionStateResult(sig),
            pool,
            1 * 1000
          );
        }
        this.lastResult = await client.transactionState(sig);
        if (this.lastResult.result === pool) {
          succeeded += 1;
          break;
        }
      }
      console.log(
        `Node ${name} response is: ${this.lastResult.result}`,
        "  ",
        succeeded,
        "of",
        total,
        "(need",
        needed,
        ")"
      );
    });
```
Sample output when one node did not get the transaction:
```